src/hawkmoth/__init__.py

```diff
@@ -360,19 +360,8 @@ def _deprecate(conf, old, new, default=None):
 def setup(app):
     app.require_sphinx('3.0')
 
-    app.add_config_value('cautodoc_root', None, 'env', [str])
-    app.add_config_value('cautodoc_clang', None, 'env', [list])
-
-    app.add_config_value(
-        'hawkmoth_root',
-        lambda conf: _deprecate(conf, 'cautodoc_root', 'hawkmoth_root', app.confdir),
-        'env', [str]
-    )
-    app.add_config_value(
-        'hawkmoth_clang',
-        lambda conf: _deprecate(conf, 'cautodoc_clang', 'hawkmoth_clang', []),
-        'env', [list]
-    )
```

Comment:
I would also delete `_deprecate` since it's no longer used. Sure it could be useful in the future, but it's so simple to add it back even if we need it in its exact form and currently it's dead code.

Reply:
I thought I did. But apparently I only thought, but didn't really. Will fix.

Reply:
Looks good!